### DISCUSSION TITLE
No definition found for 'Symbol'

### DISCUSSION THREAD

**Discussion by rosasco-wk** [redhat-developer/vscode-java#2831] [Q&A]
I'm working on a large, multi-language project on Ubuntu and am new to the Java code.  I've spent days trying to get vscode-java to recognize project symbols (specifically w/ "Go to Definition") to no avail.

The build environment for this project is Bazel.

There is a ".project" file (apparently Eclipse related) at the root of the github project.  I don't know how that affects / doesn't effect vscode-java.

I've installed OpenJDK-17 to support this plug-in.  The project itself specifies the use of JDK-11 but I think that's ok because I don't wish to build the overall project using vscode in any way.

I just want to use vscode to do some editing and code navigation to get familiar with the project.

With fairly simple setup steps, Eclipse does resolve jump-to-definition symbols for this project  That said, I don't care for using Eclipse at all for various reasons.

Any and all tips appreciated.

**Comment by SandroJijavadze** [comment]
@rosasco-wk how is it looking after a few years?  I'm dealing with the same problem.
